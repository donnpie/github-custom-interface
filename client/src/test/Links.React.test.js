import React from 'react';
import Link from '../components/Link.jsx';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer
    .create(<Link title="Link to user profile" href="http://www.facebook.com">Facebook</Link>)
    .toJSON();
    expect(tree).toMatchSnapshot();
    });