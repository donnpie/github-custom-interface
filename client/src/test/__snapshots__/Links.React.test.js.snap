// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="link-container"
>
  <a
    href="http://www.facebook.com"
  >
    Link to user profile
  </a>
</div>
`;
